doc/release/upcoming_changes/17394.new_function.rst

```diff
@@ -0,0 +1,5 @@
+``sliding_window_view`` provides a sliding window view for numpy arrays
+-----------------------------------------------------------------------
+``np.sliding_window_view`` constructs views on numpy arrays that offer a sliding
```

Comment:
```suggestion
`numpy.sliding_window_view` constructs views on numpy arrays that offer a sliding
```
so that this makes a link (`np` does not work in links)